packages/api-client/rollup.config.js

```diff
@@ -0,0 +1,107 @@
+import postcss from 'rollup-plugin-postcss';
+import autoprefixer from 'autoprefixer';
+import visualizer from 'rollup-plugin-visualizer';
+import { terser } from 'rollup-plugin-terser';
+import typescript from '@rollup/plugin-typescript';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import { babel } from '@rollup/plugin-babel';
+import replace from '@rollup/plugin-replace';
+import { defineConfig } from 'rollup';
+import pkg from './package.json';
+
+const production = !process.env.ROLLUP_WATCH;
+
+function basePlugins() {
+    return [
+        typescript({
+            sourceMap: true,
+            declaration: true,
+            declarationDir: 'dist',
+            rootDirs: ['src', 'test'],
+        }),
+        nodeResolve(),
+        commonjs(),
+        json(),
+        postcss({
+            extract: 'index.css',
+            plugins: [autoprefixer()],
+        }),
+        // Transpile to ES5 when running a production build
+        production &&
+            babel({
+                babelHelpers: 'bundled',
+                extensions: ['.ts', '.mjs', '.js', '.svelte'],
+                include: ['src/**', 'node_modules/chart.js/**', 'node_modules/svelte/**'],
+                presets: ['@babel/preset-env'],
+            }),
+    ];
+}
+
+function onwarn(warning, warn) {
+    // https://github.com/moment/luxon/issues/193
+    if (warning.code === 'CIRCULAR_DEPENDENCY') {
+        if (warning.importer.includes('node_modules/luxon')) {
+            return;
+        }
+    }
+    warn(warning);
+}
+
+const esm = defineConfig({
+    input: 'src/index.ts',
+    // Externalize all dependencies
+    external: (id) => Object.keys(pkg.dependencies).includes(id),
+    output: {
+        dir: 'dist',
+        entryFileNames: '[name].es.js',
```

Comment:
It's still expected to be named `.esm.js` in `package.json`:
```
"module": "dist/api-client.esm.js",
```
That being said, I can't find any documentation on a `module` property for package.json files, so I suppose we can just remove that line instead 😄